astropy/io/tests/safeio.py

```diff
@@ -0,0 +1,13 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import io
+
+
+class _BufferedWriter(io.BufferedWriter):
```

Comment:
If this goes in a generic test helper module it should get a more descriptive name and not be private.